mappings/net/minecraft/client/network/ClientAdvancementManager.mapping

```diff
@@ -1,8 +1,15 @@
 CLASS dgy net/minecraft/client/network/ClientAdvancementManager
+	CLASS net/minecraft/client/network/ClientAdvancementManager$a Listener
+		METHOD a setProgress (Lq;Ls;)V
+		METHOD e selectTab (Lq;)V
 	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD b client Lcvb;
 	FIELD c manager Lr;
+	FIELD d advancementProgresses Ljava/util/Map;
```

Comment:
advancementProgresses vs advancementProgressMap? What is it keyed by?

Reply:
It's an Advancement -> AdvancementProgress map.

Reply:
Right.